packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -279,20 +279,44 @@ class FuchsiaDevice extends Device {
         printError('Failed to start the Fuchsia package server');
         return LaunchResult.failed();
       }
+
+      // Serve the application's package.
       final File farArchive = package.farArchive(
           debuggingOptions.buildInfo.mode);
       if (!await fuchsiaPackageServer.addPackage(farArchive)) {
         printError('Failed to add package to the package server');
         return LaunchResult.failed();
       }
 
+      // Serve the flutter_runner.
```

Comment:
Could this use the existing `artifacts.getArtifactPath(...)` APIs instead?

Reply:
Done.